samples/HttpRequestFeaturesv2/README.md

```diff
@@ -0,0 +1,43 @@
+# About this sample
+This sample extracts HTTP request features and adds them as context features to a Rank request that can be sent to an Azure Personalizer instance. After this request is sent, the user has the option to send a followup Reward request and then regenerate a new Rank request.
```

Comment:
Minor edits. I don't understand the point of the last one - regenerate - is that taking in the reward from the previous to show how this is learning, or just convenience from the user's perspective?

```suggestion
This sample extracts features from a HTTP request and adds them as context features to a Rank request - which can be sent to an Azure Personalizer instance. Once the rank request is sent, the user can send a follow-up Reward request. If required, one can regenerate a new Rank request.
```

Reply:
The idea is, since part of the rank request is randomized, that the user can generate a new one with new information, and see  how that affects the response. Otherwise, they would only have the option of sending a single request. this behavior was the same in the old sample